AirCasting/CreateSessionViews/PowerABView.swift

```diff
@@ -23,11 +25,17 @@ struct PowerABView: View {
             }
             continueButton
                 .buttonStyle(BlueButtonStyle())
-        }
+        }.alert(isPresented: $showAlert, content: {
+            Alert(title: Text("Location alert"), message: Text("Please go to settings and allow location first!"), dismissButton: .default(Text("OK")))
```

Comment:
"Ok" button should redirect to the settings view (it's similar to when you have Bluetooth off or Bluetooth access denied).